### PR TITLE
Fix Issue 323: Refactor `CiphertextAcceptedBallot` to `Submitted Ballot`

### LINKED ISSUE #323: Refactor `CiphertextAcceptedBallot` to `Submitted Ballot`
## Issue

**Description**
Refactor `CiphertextAcceptedBallot` to `SubmittedBallot` . This is straightforward the terms should be refactored through the code base. In addition, the verbs used instead of accepted in methods should be changed to `submit`

### PR DESCRIPTION
**Pull request by OBITORASU**
_🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository._

### Issue
[Issue #323](https://github.com/microsoft/electionguard-python/issues/323)

Fixes #323 

### Description
As asked in the original issue, this PR refactors all occurances of ```CiphertextAcceptedBallot``` to ```SubmittedBallot``` in the codebase.

### CODE REVIEW

File: src/electionguard/ballot.py [hunk 1]
```diff
@@ -816,9 +816,9 @@ class BallotBoxState(Enum):
 
 
 @dataclass(unsafe_hash=True)
-class CiphertextAcceptedBallot(CiphertextBallot):
+class SubmittedBallot(CiphertextBallot):
     """
-    A `CiphertextAcceptedBallot` represents a ballot that is accepted for inclusion in election results.
+    A `SubmittedBallot` represents a ballot that is accepted for inclusion in election results.
     An accepted ballot is or is about to be either cast or spoiled.
```
Comment (VikasHanumegowda) [inline src/electionguard/ballot.py:822]: Hi @OBITORASU , "An accepted ballot" and similar occurrences should be changed to "A submitted ballot". Please update your PR :) Thanks

Comment (OBITORASU) [reply src/electionguard/ballot.py:822]: On it!

Comment (OBITORASU) [reply src/electionguard/ballot.py:822]: Just a small doubt. Do I cover all the instances of ```accepted_ballot``` as well?

Comment (VikasHanumegowda) [reply src/electionguard/ballot.py:822]: I think so, @keithrfung , please second me.

Comment (OBITORASU) [reply src/electionguard/ballot.py:822]: My commit is pretty much ready and waiting once I get the final response I will do the needful and push :)

Comment (VikasHanumegowda) [reply src/electionguard/ballot.py:822]: Cool, thanks

Comment (rc-ms) [reply src/electionguard/ballot.py:822]: thanks for jumping in folks! we're following up about your question and will get back pronto. 👍

Comment (keithrfung) [reply src/electionguard/ballot.py:822]: Sorry i'm late on the response. 

Yes. `Accepted` as a term is getting changed to `Submitted` including comments and functions.

Comment (keithrfung) [reply src/electionguard/ballot.py:822]: Also don't forget to check the linting. `make auto-lint`

**Review by keithrfung — CHANGES_REQUESTED** [review]
Held up by linting and waiting on some changes to catch all presence of `accepted`.

File: src/electionguard/ballot.py (hunk 1 above)
Comment (OBITORASU) [reply src/electionguard/ballot.py:822]: Thanks for the clarification! I will push the commit soon.

**Comment by OBITORASU** [comment]
PTAL I have pushed the requested changes :)

**Comment by keithrfung** [comment]
Ok. This is an unfortunate sequence of events but the other PR I had caused a couple conflicts with this one. Can you take a look at the conflicts?

**Comment by OBITORASU** [comment]
That should take care of the conflicting files. PTAL

**Comment by keithrfung** [comment]
I'm reviewing this today. Looks good off first glance. Thanks for the followup.

**Review by keithrfung — APPROVED** [review]
Excellent work. Welcome new contributor!

**Pull request merged**